webpack.config.js

```diff
@@ -1,174 +1,68 @@
-const fs = require('fs');
 const path = require('path');
-const webpack = require('webpack');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const HtmlBeautifyPlugin = require('@nurminen/html-beautify-webpack-plugin');
-const HtmlWebpackSkipAssetsPlugin = require('html-webpack-skip-assets-plugin').HtmlWebpackSkipAssetsPlugin;
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
-const RemoveEmptyScriptsPlugin = require('webpack-remove-empty-scripts');
-const TerserPlugin = require("terser-webpack-plugin");
+const HtmlBundlerPlugin = require('html-bundler-webpack-plugin');
 
 const isDev = process.env.WEBPACK_SERVE;
 
-function entryPoints(dir) {
-    let entry_points = {
-        style: './src/scss/style.scss'
-    };
-
-    function each_file(dir) {
-        try {
-            fs.readdirSync(dir, {withFileTypes: true}).forEach(function(item) {
-                const file = item.name;
-                const parts = file.split('.');
-                if (parts[1] !== 'js') return;
-                var file_path = dir + '/' + file;
-                entry_points[parts[0]] = file_path;
-            });
-        } catch (e) {
-            console.log(e);
-        }
-    }
-
-    each_file(dir);
-
-    return entry_points;
-}
-
-function generateHtmlPlugins(templateDir) {
-    const templateFiles = fs.readdirSync(path.resolve(__dirname, templateDir), {withFileTypes: true});
```

Comment:
The `HtmlBundlerPlugin` can automatically handle HTML templates from the [entry path](https://github.com/petrushin-ai/webpack5-project-starter/pull/1/files#option-entry-path) option.